src/components/molecules/PrevNextBar/PrevNextBar.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
```

Comment:
Needs to be mobile friendly 
![Screen Shot 2020-10-28 at 11 02 17 AM](https://user-images.githubusercontent.com/26819992/97454379-1440ea80-190d-11eb-87ca-5d85347af465.png)